nursery/create-restart-manager-session.yml

```diff
@@ -0,0 +1,9 @@
+rule:
+  meta:
+    name: create Restart Manager session
+    namespace: host-interaction/process
```

Comment:
Not sure if this is the best namespace here. At least in the context of ransomware, this resolves more around file access?

Reply:
Yeah, I was unsure where to place this rule. The Restart Manager itself can be used to stop applications so I figured throw it under `host-interaction/process` to start. However, like you said, within the context of ransomware the goal is file access, not process manipulation. 🤔